contrib/IECoreArnold/src/IECoreArnold/UniverseBlock.cpp

```diff
@@ -62,7 +62,7 @@ void loadMetadata( const std::string &pluginPaths )
 			{
 				if( dIt->path().extension() == ".mtd" )
 				{
-					if( !AiMetaDataLoadFile( dIt->path().c_str() ) )
+					if( !AiMetaDataLoadFile( (const char*)dIt->path().c_str() ) )
```

Comment:
Looking at the docs, the only reason I can think of this cast being required is if path().c_str() returns a const pointer to an array of a character type other than char - is WIndows using 16 bit characters or something?  If so, casting to char* is probably wrong.

Reply:
Hey Daniel, I get this:
Severity	Code	Description	Project	File	Line	Suppression State
Error (active)	E0167	argument of type "const boost::filesystem::path::value_type *" is incompatible with parameter of type "const char *"	IECoreArnold	c:\Users\Alex\dev\cortex\contrib\IECoreArnold\src\IECoreArnold\UniverseBlock.cpp	65